dds/DCPS/RTPS/Sedp.cpp

```diff
@@ -1335,6 +1411,30 @@ Sedp::disassociate(const ParticipantData_t& pdata)
   }
 }
 
+void
+Sedp::replay_durable_data_for(const DCPS::RepoId& remote_sub_id)
+{
+  DCPS::GuidConverter conv(remote_sub_id);
+  ACE_DEBUG((LM_DEBUG, "Sedp::replay_durable_data_for %C\n", OPENDDS_STRING(conv).c_str()));
+  if (remote_sub_id.entityId == ENTITYID_SEDP_BUILTIN_PUBLICATIONS_READER) {
+    write_durable_publication_data(remote_sub_id, false);
+  } else if (remote_sub_id.entityId == ENTITYID_SEDP_BUILTIN_SUBSCRIPTIONS_READER) {
+    write_durable_subscription_data(remote_sub_id, false);
+  } else if (remote_sub_id.entityId == ENTITYID_P2P_BUILTIN_PARTICIPANT_MESSAGE_READER) {
+    write_durable_participant_message_data(remote_sub_id);
```

Comment:
I'm not sure why there's a "write_durable" for this one (and its secure version), since the endpoints are not durable.

Reply:
It is durable in the code.  We can defer a fix.

Reply:
Yep, I'll make a note to come back to it.